tests/Tests/700__sandbox/701__performance_sandbox.robot

```diff
@@ -79,3 +79,8 @@ Verify Sandbox Toolchain Data
                   ...   expected in toolchain result=> ${km_data[0]} : ${m_value}
            END
     END
+
+Performance Suite Setup
+    #Oc Apply    kind=OLMConfig    src=${CURDIR}/olm.yaml   #Not working
```

Comment:
[Documentation] missing(explaining that  we are disabling CopiedCSVs and why)

Reply:
Done